client/stylesheets/_toCat.scss

```diff
@@ -2547,6 +2551,14 @@ i:hover {
     z-index: 2;
 }
 
+.headerPreDrop {
+    margin-left: -172px !important;
+}
+
+.Shoes {
+    margin-left: -192px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces